### ISSUE TITLE
mac os 360 rot_reset error

### ISSUE THREAD

**Issue by cispadmin** [dfaker/VR-reversal#30]
Hello! When I tried to run your script on my macbook I received the below error. MacOs Big Sur. Macbook Pro late 2013.
ffmpeg installed.
Can you tell me how to fix this situation?
Thank you in advance for your help and time!

mpv --script=~/.config/mpv/360plugin.lua --script-opts=360plugin-enabled=yes --hwdec=yes --vf=lavfi="[minterpolate=fps=60000/1001:mi_mode=mci]" ~/Video/video.mp4
Passing more than 1 argument to vf-add is deprecated!
 (+) Video --vid=1 (*) (h264 1920x1080 50.000fps)
 (+) Audio --aid=1 (*) (aac 2ch 44100Hz)
[lavfi] AVOption 'reset_rot' not found.
[user_filter_wrapper] Creating filter 'v360' failed.
Video: no video
Exiting... (Errors when loading file)

**Comment by dfaker** [comment]
You're using an old version of mpv from prior to the introduction of the reset_rot option on the [v360](https://ffmpeg.org/ffmpeg-filters.html#toc-v360) filter.

**Comment by cispadmin** [comment]
Version mpv on github - 
RELEASE_NOTES | Release 0.37.0
-- | --
[RELEASE_NOTES](https://github.com/mpv-player/mpv/blob/master/RELEASE_NOTES)
[Release 0.37.0](https://github.com/mpv-player/mpv/commit/818ce7c51a6b9179307950e919983e0909942098)

This is my version:
mpv 0.37.0 Copyright © 2000-2023 mpv/MPlayer/mplayer2 projects
 built on Mar 14 2024 15:36:06
libplacebo version: v6.338.1
FFmpeg version: 4.4.4
FFmpeg library versions:
   libavutil       56.70.100
   libavcodec      58.134.100
   libavformat     58.76.100
   libswscale      5.9.100
   libavfilter     7.110.100
   libswresample   3.9.100

**Comment by dfaker** [comment]
yeah there was some dancing with how v360 worked in that version that wasn't accommodated until ffmpeg 5.0 and later.

check one of the versions at https://mpv.io/installation/ specifically the recent ones here: https://laboratory.stolendata.net/~djinn/mpv_osx/
